exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/ExporterMetrics.java

```diff
@@ -70,4 +70,9 @@ public static ExporterMetrics createGrpcOkHttp(String type, MeterProvider meterP
   public static ExporterMetrics createHttpProtobuf(String type, MeterProvider meterProvider) {
     return new ExporterMetrics(meterProvider.get("io.opentelemetry.exporters.otlp-http"), type);
   }
+
+  public static ExporterMetrics createHttpJson(String type, MeterProvider meterProvider) {
```

Comment:
Since this is only really for testing anyways, we probably don't need a separate metrics for it